src/fabglconf.h

```diff
@@ -127,7 +127,8 @@
 
 
 /** Size (in bytes) of largest block to maintain free */
-#define FABGLIB_MINFREELARGESTBLOCK 40000
+// #define FABGLIB_MINFREELARGESTBLOCK 40000
+#define FABGLIB_MINFREELARGESTBLOCK 4000
```

Comment:
this change helps ensure that the VDP will remain fully operational as memory fragments

in reality we do not need to ensure we have a minimum sized free block of 40000 bytes.  the largest contiguous block size we should practically need is for around two rows of the screen pixels.  as our frame buffers are split into separately allocated rows there's no real need to look for a larger contiguous block.  doing so could prevent changing into higher memory requirement screen buffers after having used other parts of the VDP (such as buffers)